src/dash-table/components/Table/Table.less

```diff
@@ -260,7 +254,7 @@
             &.cell--selected {
                 background-color: var(--selected-background);
             }
-
+            
```

Comment:
I think we can slice out that one ☝️

There's also an entry in `Dropdown.css` that can be removed (the `border-radius` is redundant anyway as the `border: none` is applied elsewhere in that file..

```
.dash-spreadsheet .cell--selected .Select-control {
    background-color: var(--selected-background);
    border-radius: 0;
}
```